README.md

```diff
@@ -2,37 +2,146 @@
 
 The JS Buy SDK is a lightweight library that allows you to build ecommerce into
-any website. It's based on Shopify's JSON API and provides the ability to
+any website. It's based on Shopify's GraphQL API and provides the ability to
 retrieve products and collections from your shop, add products to a cart, and
 checkout.
 
-If you're just getting started, please [read the docs](http://shopify.github.io/js-buy-sdk/).
+Docs are still a work in progress, but you can view the [API docs](https://github.com/Shopify/js-buy-sdk/blob/v1.0alpha/docs).
 
-## Example
+## Table Of Contents
+
+- [Installation](#installation)
+- [Builds](#builds)
+- [Examples](#examples)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Installation
+**With Yarn:**
+```bash
+$ yarn add shopify-buy@alpha
+```
+**With NPM:**
+```bash
+$ npm install shopify-buy@alpha
+```
+
+## Builds
+The JS Buy SDK has four build versions: ES, CommonJS, AMD, and UMD.
+
+**ES, CommonJS:**
+```javascript
+import Client from 'shopify-buy';
+```
+**AMD:**
+```javascript
+import Client from 'shopify-buy/index.amd';
+```
+**UMD:**
 ```javascript
-const shopClient = ShopifyBuy.buildClient({
-  accessToken: 'bf081e860bc9dc1ce0654fdfbc20892d',
-  appId: 6,
-  domain: 'embeds.myshopify.com'
+import Client from 'shopify-buy/index.umd';
+```
+
+## Examples
+
+### Initializing the Client
+```javascript
+import Client, {Config} from 'shopify-buy';
+
+const config = new Config({
+  domain: 'your-shop-name.myshopify.com',
+  storefrontAccessToken: 'your-storefront-access-token'
 });
 
-shopClient.fetchAllProducts()
-.then(function (products) {
+const client = new Client(config);
+```
+
+### Fetching Products
+```javascript
+// Fetch all products in your shop
+client.fetchAllProducts().then((products) => {
+  // Do something with the products
   console.log(products);
-})
-.catch(function () {
-  console.log('Request failed');
+});
+
+// Fetch a single product by ID
+client.fetchProduct('123456').then((product) => {
+  // Do something with the product
+  console.log(product);
+});
+```
+
+### Fetching Collections
+```javascript
+// Fetch all collections, including their products
+client.fetchAllCollectionsWithProducts().then((collections) => {
+  // Do something with the collections
+  console.log(collections);
+  console.log(collections[0].products);
+});
+
+// Fetch a single collection by ID, including its products
+client.fetchCollectionWithProducts('123456').then((collection) => {
+  // Do something with the collection
+  console.log(collection);
+  console.log(collection.products);
+});
+```
+
+### Creating, Adding and Removing Items from a Checkout
+```javascript
+// Create an empty checkout
+client.createCheckout().then((checkout) => {
+  const addLineItemsInput = {
+    checkoutId: checkout.id,
+    lineItems: [
+      {variantId: 'gid://shopify/ProductVariant/12345', quantity: 5}
```

Comment:
I think that only happens because forms convert their input values to strings, so rather than `5` it sends in `"5"`. If we're passing in just numbers, it should work without `parseInt`.